### DISCUSSION TITLE
Apple Silicon support?

### DISCUSSION THREAD

**Discussion by ballo** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#334] [General]
Hello. I've been looking for a solution to use an external adapter for my M2 MacBook. Has anyone tried to use this driver on Apple Silicon Macs? My other idea is to use my nvme adapter with a M-key to M.2 adaper and one of the Apple Broadcom cards. The KEXTs for these cards exist, although I don't know where the kmod binaries are to see if they are indeed compiled for ARM.

**Comment by SB99-07** [comment]
I would also like to know?

**Comment by EAZYBLACK** [comment]
you cant
